modules/MusicDownloader.js

```diff
@@ -125,6 +176,14 @@ async function getTrackInfo(url) {
         logger('log', 'Wykryto piosenkę', 'getTrackInfo');
         return await spotify.getTrack(url);
     }
+    } catch (e) {
+        logger('error', "Błąd w trakcie wykonywania funkcji downloadSong", 'getTrackInfo');
+        logger('error', e, 'getTrackInfo');
+        if (global.debugmode === true) {
+            DebugSaveToFile('MusicDownloader', 'getTrackInfo', 'error_main_function', e);
```

Comment:
Zmień error_main_function na catched_error
```suggestion
            DebugSaveToFile('MusicDownloader', 'getTrackInfo', 'catched_error', e);
```